src/bindings/kotlin/gen_kotlin.rs

```diff
@@ -283,13 +284,13 @@ internal interface _UniFFILib : Library {
 // Public interface members begin here.
 
 {% for e in ci.iter_enum_definitions() %}
-    enum class {{ e.name() }} {
+    enum class {{ e.name()|decl_name_kt }} {
```

Comment:
It's probably not clear from the code, but in my head I'd originally intended the "decl" prefix on these filter function to be read as "declare type". Something like "class_name_kt" might be a clearer name for this filter (and the existing "decl_kt" might be better as something like "type_kt" for clarity).

Reply:
Ok, I'll change functions as you suggest.

Reply:
:+1: